acceptance/suites/pre_suite/puppet3_compat/30_install_dev_repos.rb

```diff
@@ -6,11 +7,10 @@
   end
 
   step "Setup Puppet Server dev repository on the master." do
-    install_opts = options.merge({ :dev_builds_repos => ["PC1"]})
     install_puppetlabs_dev_repo(master, 'puppetserver', package_build_version, nil, install_opts)
   end
 
   step "Setup Puppet dev repository on the master." do
-    install_puppet_agent_dev_repo_on(master, {:puppet_agent_version => test_config[:puppet_build_version]})
+    install_puppetlabs_dev_repo(host, 'puppet-agent', test_config[:puppet_build_version], nil, install_opts)
```

Comment:
I missed that the first parameter to this call should have been `master` instead of `host`.  This line caused the puppet3_compat tests to fail.  I put up #1402 to address this.